reaktive/src/nativeCommonMain/kotlin/com/badoo/reaktive/observable/ToList.kt

```diff
@@ -0,0 +1,19 @@
+package com.badoo.reaktive.observable
+
+import com.badoo.reaktive.single.Single
+import kotlin.native.concurrent.isFrozen
+
+actual fun <T> Observable<T>.toList(): Single<List<T>> =
+    collect(ArrayList()) { list, item ->
+        val newList =
+            if (list.isFrozen) {
```

Comment:
If `list.isFrozen` is always false, then we will have following behaviour:
```kotlin
val newList = list
newList.add(item)
newList
```

Which is absolutely the same as `jvmJsCommon` implementation. Merge them?